src/Estimators/tests/test_SpinDensityNew.cpp

```diff
@@ -16,7 +16,7 @@
 #include "ValidSpinDensityInput.h"
 #include "SpinDensityNew.h"
 #include "RandomForTest.h"
-#include "Utilities/SpeciesSet.h"
+//#include "Utilities/SpeciesSet.h"
```

Comment:
should be able to just delete now.

Reply:
Will handle it when I deal with merge conflicts.